include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -363,6 +358,15 @@ struct reduce_over_group
     {
         __init_processing<_Tp>{}(__init, __result, __bin_op1);
     }
+
+    inline ::std::size_t
+    local_mem_req(const ::std::uint16_t& __work_group_size) const
+    {
+        if (__has_known_identity<_BinaryOperation1, _Tp>{})
```

Comment:
Not super important, but this can be an `if constexpr`.

Reply:
done